src/server.rs

```diff
@@ -84,7 +84,9 @@ serializable_enum!(ResponseData,
     TextEdit([TextEdit; 1]),
     Locations(Vec<Location>),
     Highlights(Vec<DocumentHighlight>),
-    HoverSuccess(Hover)
+    HoverSuccess(Hover),
+    // Empty tuple is represented as null in JSON
+    Null(())
```

Comment:
I believe the protocol leaves it undefined - i.e., we can do what we want. Not that we should send a value `undefined`, since aiui such a thing doesn't exist.